modules/ipi-install-creating-a-disconnected-registry.adoc

```diff
@@ -5,254 +5,70 @@
 ----
+<1> You can avoid running `--tls-verify=false` by configuring your system to trust the generated rootCA certificates. See "Using SSL to protect connections to Red Hat Quay" and "Configuring the system to trust the certificate authority" for more information.
 +
-For `<local_registry_host_name>`, specify the registry domain name for your mirror
-repository, and for `<local_registry_host_port>`, specify the port that it
-serves content on.
-
-.. Set the local repository name:
-+
-[source,terminal]
-----
-$ LOCAL_REPO='<local_repository_name>'
-----
-+
-For `<local_repository_name>`, specify the name of the repository to create in your
-registry, such as `ocp4/openshift4`.
-
-. Mirror the remote install images to the local repository.
-+
-[source,terminal]
-----
-$ /usr/local/bin/oc adm release mirror \
-  -a pull-secret-update.txt \
-  --from=$UPSTREAM_REPO \
-  --to-release-image=$LOCAL_REG/$LOCAL_REPO:${VERSION} \
-  --to=$LOCAL_REG/$LOCAL_REPO
-----
-
-== Modify the `install-config.yaml` file to use the disconnected registry (optional)
-
-On the provisioner node, the `install-config.yaml` file should use the newly created pull-secret from the `pull-secret-update.txt` file. The `install-config.yaml` file must also contain the disconnected registry node's certificate and registry information.
-
-.Procedure
-
-. Add the disconnected registry node's certificate to the `install-config.yaml` file. The certificate should follow the `"additionalTrustBundle: |"` line and be properly indented, usually by two spaces.
-+
-[source,terminal]
-----
-$ echo "additionalTrustBundle: |" >> install-config.yaml
-$ sed -e 's/^/  /' /opt/registry/certs/domain.crt >> install-config.yaml
-----
+[NOTE]
+====
+You can also log in by accessing the UI at `\https://<host.example.com>:8443` after installation.
+====
 
-. Add the mirror information for the registry to the `install-config.yaml` file.
+. You can mirror {product-title} images after logging in. Depending on your needs, see either the "Mirroring the {product-title} image repository" or the "Mirroring Operator catalogs for use with disconnected clusters" sections of this document.
 +
-[source,terminal]
-----
-$ echo "imageContentSources:" >> install-config.yaml
-$ echo "- mirrors:" >> install-config.yaml
-$ echo "  - registry.example.com:5000/ocp4/openshift4" >> install-config.yaml
-$ echo "  source: quay.io/openshift-release-dev/ocp-release" >> install-config.yaml
-$ echo "- mirrors:" >> install-config.yaml
-$ echo "  - registry.example.com:5000/ocp4/openshift4" >> install-config.yaml
-$ echo "  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev" >> install-config.yaml
-----
-+
-NOTE: Replace `registry.example.com` with the registry's fully qualified domain name.
+[NOTE]
+====
+If there are issues with images stored by the _mirror registry for Red Hat OpenShift_ due to storage layer problems, you can remirror the {product-title} images, or reinstall mirror registry on more stable storage.
```

Comment:
> Avoid using expletive constructions such as it is, there are, and there is, which hide the subject of the sentence.

If you know what kind of issues, it would be better to say that. Like, is this here because of errors during installation?

```suggestion
If there are issues with images stored by the _mirror registry for Red Hat OpenShift_ due to storage layer problems, you can remirror the {product-title} images or reinstall mirror registry on more stable storage.
```